### DISCUSSION TITLE
This repository is active?

### DISCUSSION THREAD

**Discussion by morad** [MacPaw/OpenAI#136] [General]
Hello,

Myself and few others are more than happy to jump in and assist where needed to get this repo up to date with the latest of OpenAI API specs.

**Comment by Krivoblotsky** [comment]
Hey, @morad!
Yes, it is absolutely active and has a great community.
Feel free to contribute, it would be awesome 🤩

**Comment by kalafus** [comment]
@morad , up to date with the latest API specs as of [PR 169](https://github.com/MacPaw/OpenAI/pull/169)
for beta support of [assistants endpoints](https://github.com/MacPaw/OpenAI/pull/140)
